examples/data-transformations.yaml

```diff
@@ -0,0 +1,100 @@
+# See doc docs/data-sourcing-and-transformation.md
+apiVersion: argoproj.io/v1alpha1
```

Comment:
nice, can we make this a test with `workflows.argoproj.io/test`? please use e2e whitelisted images if you do that (this isn't enforced for `make test-examples`)

Reply:
Sure, will do

Reply:
bump

Reply:
Just took another look. I don't think this is a good candidate for `workflows.argoproj.io/test` yet. This test requires a starting state in a Minio repository (a bucket with artifacts in it). Not sure if adding that behavior to an example will be in line with the example's purpose: to simply show how the template works.

There is an e2e test already present that does this and essentially tests the contents of this example while also generating its own data.

Reply:
sure, we can change our mind later